### PR TITLE
Story/quad 16

### PR DESCRIPTION
**Pull request by HariChandanaKanchanapally**

### CODE REVIEW

File: Quadriga/src/main/java/edu/asu/spring/quadriga/web/workspace/ActivateWSController.java [hunk 1]
```diff
@@ -57,114 +54,49 @@ protected void initBinder(WebDataBinder binder) {
...
+	 * @throws QuadrigaStorageException
+	 * @throws QuadrigaAccessException
+	 * @author Hari Chandana Kanchanapally
+	 */
     @AccessPolicies({
-            @ElementAccessPolicy(type = CheckedElementType.PROJECT, paramIndex = 1, userRole = {
+            @ElementAccessPolicy(type = CheckedElementType.WORKSPACE, paramIndex = 1, userRole = {
                     RoleNames.ROLE_COLLABORATOR_ADMIN,
                     RoleNames.ROLE_PROJ_COLLABORATOR_ADMIN,
                     RoleNames.ROLE_PROJ_COLLABORATOR_CONTRIBUTOR }),
```
Comment (jdamerow) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/web/workspace/ActivateWSController.java:73]: I believe those roles are wrong. They should be the corresponding workspace roles.

File: Quadriga/src/main/java/edu/asu/spring/quadriga/web/workspace/ActivateWSController.java [hunk 2]
```diff
@@ -57,114 +54,49 @@ protected void initBinder(WebDataBinder binder) {
...
-        ModelAndView model;
-        List<ModifyWorkspace> activateWSList;
+            @PathVariable("workspaceid") String workspaceid,
+            @RequestParam("projectid") String projectid, Principal principal,
+            RedirectAttributes redirectAttributes)
+            throws QuadrigaStorageException, QuadrigaAccessException {
 
+        String userName;
         // fetch the user name
         userName = principal.getName();
```
Comment (jdamerow) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/web/workspace/ActivateWSController.java:87]: Merge this line with line 85.

**Comment by jdamerow** [comment]
see comments

File: Quadriga/src/main/java/edu/asu/spring/quadriga/web/workspace/ActivateWSController.java (hunk 1 above)
Comment (HariChandanaKanchanapally) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/web/workspace/ActivateWSController.java:73]: Added the workspace roles to Workspace Collaborator Admin and Contributor. Did not add the  ROLE_PROJ_COLLABORATOR_EDITOR as it is for the ListController which is longer needed.

File: Quadriga/src/main/java/edu/asu/spring/quadriga/web/workspace/ActivateWSController.java (hunk 2 above)
Comment (HariChandanaKanchanapally) [inline Quadriga/src/main/java/edu/asu/spring/quadriga/web/workspace/ActivateWSController.java:87]: Merged for declaration and initialisation on the same line

**Pull request closed without merging**